Libraries/test/EventsTests.Shared/EventTests.cs

```diff
@@ -482,6 +482,23 @@ public void DynamoDbUpdateTest(Type serializerType)
             var recordDateTime = record.Dynamodb.ApproximateCreationDateTime;
             Assert.Equal(recordDateTime.Ticks, 636162388200000000);
 
+            var topLevelList = record.Dynamodb.NewImage["misc1"].L;
```

Comment:
In `Libraries/test/EventsTests.Shared/dynamodb-event.json`, I added properties for all possible `AttributeValue` (to confirm they could be deserialized correctly).

---

I also noticed the xUnit signature is `Equal<T>(T expected, T actual)`, but most of the existing tests instead check `Equal(actual, expected)`... I followed the correct format here, but did not change the existing assert statements.